### PR TITLE
add new Fib example with many chunks

### PR DESCRIPTION
**Pull request by leonardoalt**

### CODE REVIEW

File: examples/multiple_chunks/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        .out_path("powdr-target")
+        .chunk_size_log2(18)
+        .build()
+        .write(1, &data)
+        .write(2, &data.iter().sum::<u32>());
+
+    // Fast dry run to test execution.
+    session.run();
+
+    // Uncomment to compute the proof.
```
Comment (Schaeff) [inline examples/multiple_chunks/src/main.rs:19]: it's uncommented

File: riscv/src/continuations.rs [hunk 2]
```diff
@@ -141,6 +146,11 @@ where
                     ),
                 ]);
                 pipeline_callback(pipeline)?;
+
+                if let Some(original_dir) = parent_dir {
+                    pipeline.set_output(original_dir, force_overwrite);
+                }
```
Comment (leonardoalt) [inline riscv/src/continuations.rs:152]: Before this fix continuations would do:

```console
[2024-11-01T09:43:02Z INFO  powdr_pipeline::pipeline] Writing powdr-target/chunk_0/chunk_1/guest_proof.bin.                                                   
```

Comment (leonardoalt) [reply riscv/src/continuations.rs:152]: Now:
```console
Writing powdr-target/chunk_1/guest_proof.bin.
```

File: examples/multiple_chunks/src/main.rs (hunk 1 above)
Comment (leonardoalt) [reply examples/multiple_chunks/src/main.rs]: removed comment

**Comment by leonardoalt** [comment]
After talking to @Schaeff  I'll add a different example where the guest has some explicit computation that span across multiple chunks.

**Review by Schaeff — APPROVED** [review]

**Pull request merged**